odbc_diag_record.c

```diff
@@ -511,12 +511,37 @@ odbc_get_diag_field (SQLSMALLINT handle_type,
 	case SQL_DIAG_CONNECTION_NAME:	/* yet not implemeted */
 	  if (diag_info_ptr && buffer_length > 0)
 	  {
+	    int connhd = -1;
+		int cci_ret = 0;
+	    T_CCI_ERROR cci_err_buf;
+
 		*((char *) diag_info_ptr) = '\0';

```

Comment:
I think this line is not required. Either diag_info_ptr shall be filled by 'cci_get_cas_info' or set to '\0' in line #537.